datafusion/core/tests/physical_optimizer/enforce_distribution.rs

```diff
@@ -453,7 +453,7 @@ impl TestConfig {
         &self,
         expected_lines: &[&str],
         plan: Arc<dyn ExecutionPlan>,
-        optimizers_to_run: Vec<Run>,
+        optimizers_to_run: &[Run],
```

Comment:
this is the core API change and the rest of the PR just updated the code to compile